specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/scheduledActions.json

```diff
@@ -0,0 +1,940 @@
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DaysOfWeek": {
+      "type": "string",
+      "description": "Days of Week.",
+      "enum": [
+        "Monday",
+        "Tuesday",
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday",
+        "Sunday"
+      ],
+      "x-ms-enum": {
+        "name": "DaysOfWeek",
+        "modelAsString": true
+      }
+    },
+    "ErrorResponse": {
+      "description": "Error response indicates that the service is not able to process the incoming request. The reason is provided in the error message. \n\nSome Error responses: \n\n * 429 TooManyRequests - Request is throttled. Retry after waiting for the time specified in the \"x-ms-ratelimit-microsoft.consumption-retry-after\" header. \n\n * 503 ServiceUnavailable - Service is temporarily unavailable. Retry after waiting for the time specified in the \"Retry-After\" header.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The details of the error.",
+          "$ref": "#/definitions/ErrorDetails"
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "The details of the error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "FileDestination": {
+      "type": "object",
+      "description": "Destination of the view data. This is optional. Currently only csv format is supported.",
+      "properties": {
+        "fileFormats": {
+          "type": "array",
+          "description": "Destination of the view data. Currently only csv format is supported.",
+          "items": {
+            "$ref": "#/definitions/FileFormat"
+          }
+        }
+      }
+    },
+    "FileFormat": {
+      "type": "string",
+      "description": "Destination of the view data. Currently only csv format is supported.",
+      "enum": [
+        "Csv"
+      ],
+      "x-ms-enum": {
+        "name": "FileFormat",
+        "modelAsString": true
+      }
+    },
+    "Language": {
+      "type": "string",
+      "description": "Language in which the email is localized, for ex: 'Fr'. When the specified language is not supported, it will be defaulted to english. The supported languages are cs (Czech), da (Danish), de (German), en (English), es (Spanish), fr (French), hu (Hungarian), id (Indonesian), it (Lithuanian), ja (Japanese), ko (Korean), nb (Norwegian Bokmål), nl (Dutch), pl (Polish), pt (Portuguese), ru (Russian), sv (Swedish), tr (Turkish), zh (Chinese) "
```

Comment:
>When the specified language is not supported, it will be defaulted to english. [](http://example.com/codeflow?start=78&length=78)

Does this basically mean 'When an email cannot be localized for the specified language, or the language is not currently supported, email will be sent in English'?

Reply:
Correct. I will update the description for better clarity.